tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -4459,6 +4482,25 @@ public function createCalendarEntriesWithRoom(string $user, string $identifier,
 		$this->assertStatusCode($this->response, 200);
 	}
 
+	/**
+	 * @param string $user
+	 * @param string $identifier
+	 * @param int $statusCode
+	 * @param string $apiVersion
+	 * @param TableNode|null $formData
+	 * @return void
+	 *
+	 * @Given /^user "([^"]*)" creates calendar events inviting user "([^"]*)" \((v4)\)$/
+	 */
+	public function createEventsForOneToOne(string $user, string $participant, string $apiVersion = 'v1', ?TableNode $formData = null): void {
+		$this->sendRequest('POST', '/apps/spreedcheats/mutualEvents', [
```

Comment:
```suggestion
		$this->setCurrentUser($user);
		$this->sendRequest('POST', '/apps/spreedcheats/mutualEvents', [
```

Otherwise admin creates the calendar entry instead of participant1